stix_shifter_modules/msatp/stix_translation/json/to_stix_map.json

```diff
@@ -735,22 +783,37 @@
         "references": "fc_process_file"
       }
     ],
-    "ComputerName": {
-      "key": "x_msatp.computer_name",
-      "cybox": false
-    },
-    "MachineId": {
-      "key": "x_msatp.machine_id",
-      "cybox": false
+    "DeviceId": {
+      "key": "x-oca-asset.id",
```

Comment:
change this to `x-oca-asset.deviceid`. stix-validator throws invalid object errors

`id` is standard stix property so it has to match certain regex pattern.

Reply:
fixed